docs/advanced/monitoring.md

```diff
@@ -4,56 +4,40 @@ description: Add monitoring credentials to help the Obol Team monitor the health
 
-## Import Pre-Configured Charon Dashboards
+| Alert Name | Description | Trouble shoot |
+| --- | --- | --- |
+| ClusterBeaconNodeDown | This alert is activated when the beacon node in a the cluster is offline. The beacon node is crucial for validating transactions and producing new blocks. Its unavailability could disrupt the overall functionality of the cluster | Most likely data is corrupted. Wipe data from the point you know data was corrupted and restart beacon node so it can be synced again. |
+| ClusterMissedAttestations | This alert indicates that there have been missed attestations in the cluster. Missed attestations may suggest that validators are not operating correctly, compromising the security and efficiency of the cluster. | This alert is triggered when 3 attestation are missed in 2 minutes. Check if threshold peers are online. If correct, check beacon node api error and downstream validator errors using Loki. Lastly, debug the docker  using docker compose debug. |
+| ClusterInUnknownStatus |  This alert is designed to activate when a node within the cluster is detected to be in an unknown state. The condition is evaluated by checking whether the maximum of the app_monitoring_readyz metric is 0. | This is most likely a bug in charon. report to us via https://discord.com/channels/849256203614945310/970759460693901362. |
+| ClusterInsufficientPeers | This alert is set to activate when the number of peers for a node in the cluster is insufficient. The condition is evaluated by checking whether the maximum of the app_monitoring_readyz equals 4. | If you are running group cluster, check with other peers to troubleshoot issue. If you are running solo cluster, look into other machines running the DVs to find the problem, |
+| ClusterFailureRate | This alert is activated when the failure rate of the cluster exceeds a certain threshold, more specifically, more than 5% failures in duties in the last 6 hours. | Check the upstream and downstream dependencies, latency and hardware issues. |
+| ClusterVCMissingValidators | This alert is activated if any validators in the cluster are missing. This happens when validator client cannot load validator keys in past 10 minutes. | Find if validator keys are missing and load them. |
+| ClusterHighPctFailedSyncMsgDuty |  This alert is activated if a high percentage of sync message duties failed in the cluster. The alert is activated if the sum of the increase in failed duties tagged with "sync_message" in the last hour divided by the sum of the increase in total duties tagged with "sync_message" in the last hour is greater than 0.1. | There is no troubleshoot for this. report it to us via https://discord.com/channels/849256203614945310/970759460693901362. |
```

Comment:
maybe something like
```suggestion
| ClusterHighPctFailedSyncMsgDuty |  This alert is activated if a high percentage of sync message duties failed in the cluster. The alert is activated if the sum of the increase in failed duties tagged with "sync_message" in the last hour divided by the sum of the increase in total duties tagged with "sync_message" in the last hour is greater than 10%. | This may be due to limitations in beacon node performance on nodes within the cluster. In charon, this duty is the most demanding, however an increased failure rate does not impact rewards. |
```